imports/react-ui/settings/knowledgebase/containers/article/list.js

```diff
@@ -0,0 +1,34 @@
+import { compose } from 'react-komposer';
+import { getTrackerLoader, composerOptions } from '/imports/react-ui/utils';
+import { Meteor } from 'meteor/meteor';
+import { KbArticles, KbCategories } from '/imports/api/knowledgebase/collections';
+import { pagination } from '/imports/react-ui/common';
+import { KbArticleList } from '../../components';
+
+function articlesComposer({ queryParams }, onData) {
+  const { limit, loadMore, hasMore } = pagination(queryParams, 'kb_articles.list.count');
+  const kbArticlesHandler = Meteor.subscribe(
+    'kb_articles.list',
+    Object.assign(queryParams, { limit }),
+  );
+  const kbCategoriesHandler = Meteor.subscribe('kb_categories.list', Object.assign({}, { limit }));
```

Comment:
Don't use `Object.assign()` functions if it's not needed.

Reply:
the whole line was removed after refactoring